src/goose/cli/main.py

```diff
@@ -73,11 +74,31 @@ def session_start(profile: str, plan: Optional[str] = None) -> None:
             _plan = yaml.load(f)
     else:
         _plan = None
-
     session = Session(profile=profile, plan=_plan)
     session.run()
 
 
+def parse_params(ctx: click.Context, param: click.Parameter, value: str) -> Dict[str, str]:
+    if not value:
+        return {}
+    params = {}
+    for item in value.split(","):
+        key, val = item.split(":")
+        params[key.strip()] = val.strip()
+
+    return params
+
+
+@session.command(name="planned")
+@click.option("--plan", type=click.Path(exists=True))
+@click.option("-p", "--params", callback=parse_params, help="Parameters in the format param1:value1,param2:value2")
```

Comment:
i can see `-p` being confusing for plan and params... maybe `-a/--args` instead? up to you tho!

Reply:
yes, makes sense! agree